src/routes/(default)/+page.svelte

```diff
@@ -29,17 +43,25 @@
   .index--container {
     display: flex;
     flex-direction: column;
+    height: 100%;
+    width: 75vw;
   }
   .top--container {
     display: flex;
     flex-direction: column;
-    padding: 4rem 4rem 0rem 4rem;
+    padding: 4rem 0rem 0rem 4rem;
```

Comment:
```suggestion
    padding: 4rem 0 0 4rem;
```
Mais ce padding me semble étrange (top et left mais pas right ?)

Reply:
On a oublié en bas en effet pas right parce que la liste des liens va jusqu'au bout dans les maquettes